terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1088,18 +1088,13 @@ def sq(*words):
                                       to_port=4)  # Fragmentation required DF-flag set
                 ],
                 'ingress': [
-                    vpc.security_rule(description='Any traffic from the '
-                                                  f"{'VPC' if split_tunnel else 'internet'}",
-                                      cidr_blocks=[
-                                          '${aws_vpc.gitlab.cidr_block}'
-                                          if split_tunnel else
-                                          all_ipv4
-                                      ],
+                    vpc.security_rule(description='Any traffic from the VPC',
+                                      cidr_blocks=['${aws_vpc.gitlab.cidr_block}'],
                                       protocol=-1,
                                       from_port=0,
                                       to_port=0),
                     vpc.security_rule(description='ICMP for PMTUD',
-                                      cidr_blocks=[all_ipv4],
+                                      cidr_blocks=['${aws_vpc.gitlab.cidr_block}'],
```

Comment:
Let's remove this change. We don't get flagged for it on the other security groups and it would be odd to be inconsistent.